pycue/opencue/wrappers/allocation.py

```diff
@@ -13,18 +13,20 @@
 #  limitations under the License.
 
 
-
 """
 allocation module
 """
 
-import host
-import subscription
 
 from opencue.compiled_proto import facility_pb2
 from opencue.compiled_proto import host_pb2
 from opencue.cuebot import Cuebot
-
+try:
```

Comment:
Can we not use the absolute imports here?

Reply:
Done. The only other option I found that worked was absolute imports w/ full namespaces, and I was trying to avoid updating the code to make that work, but it turns out it wasn't that bad and the code looks better as a result too.